src/mdns.rs

```diff
@@ -0,0 +1,444 @@
+use esp_idf_sys::*;
+
+extern crate alloc;
+use alloc::vec::Vec;
+
+use std::ffi::CStr;
```

Comment:
Please use the module of esp_idf_svc itself, which exposes either STD or non-std ffi support

Reply:
We Fixed it.